map.py

```diff
@@ -83,15 +82,11 @@ def generate_mapping_information(num_clients: int) -> tuple[nx.MultiDiGraph, int
     client_subset = random.sample(list(graph_copy.nodes), num_clients)
 
     for node_id in client_subset:
-        map_network.nodes[node_id]["demand_water"] = random.choice([1, 2])
-        map_network.nodes[node_id]["demand_food"] = random.choice([1, 2])
-        map_network.nodes[node_id]["demand_other"] = random.choice([1, 2])
+        map_network.nodes[node_id]["demand"] = 0
 
-        map_network.nodes[node_id]["demand"] = (
-            map_network.nodes[node_id]["demand_water"]
-            + map_network.nodes[node_id]["demand_food"]
-            + map_network.nodes[node_id]["demand_other"]
-        )
+        for i in range(len(RESOURCES)):
+            map_network.nodes[node_id][f"resource_{i}"] = random.choice([1, 2])
+            map_network.nodes[node_id]["demand"] += map_network.nodes[node_id][f"resource_{i}"]
```

Comment:
(not for this PR) We could also allow for changing the relative number of resources and add comparable units for the resources. Also allowing for capacities to depend on the resource (e.g., 1 truck can fit 800l of water, but only 300kg of food, or something like that).

Reply:
Yeah I also had the idea of adding a magnitude or range for each resource in the config file, because 200 of a food item makes sense but if it's changed to 200 people or 200 large animals or some other thing a truck could carry, 200 doesn't make sense anymore.

Reply:
Something to keep in mind for future updates. 🙂

Reply:
I did a quick kind of implementation of this, lemme know what you think https://github.com/dwave-examples/mvrp/pull/11/commits/b87cd34c7b40ca88d095e4bf9107cb5c7e87060d

Reply:
Looks good!